packages/oidc-middleware/package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "test": "yarn test:unit && yarn test:e2e && yarn test:integration",
     "test:e2e": "webdriver-manager update --gecko false && protractor test/e2e/conf.js",
-    "test:integration": "sh tools/runIntegrationTests.sh",
+    "test:integration": "/bin/bash tools/runIntegrationTests.sh",
```

Comment:
we should probably just make these files executable, and then trigger it with `./tools/runIntegrationTests.sh` (then we could change the actual shell anytime? 🤔 )

Reply:
I tried doing that but it still failed on travis. 
N00b question - If I do `chmod +x tools/runIntegrationTests.sh` and then checkin the files, do the files retain permission in github repo?

Reply:
I tried this and it still failed - 
`"test:integration": "chmod +x tools/runIntegrationTests.sh && ./tools/runIntegrationTests.sh",`